changelog/unreleased/issue-4583

```diff
@@ -0,0 +1,7 @@
+Bugfix: Ignoring the s3.storage-class option for metadata when archive tier is specified
+
+Restic now will save snapshot metadata to non-archive storage tier whatsoever,
+thus snapshot metadata could be accessed from machine without local cache.
+
+https://github.com/restic/restic/issues/4583
+https://github.com/restic/restic/issues/3202
```

Comment:
This PR does not add full blown support for archive storage. As such it should be clear after reading that there is _no_ official support for cold storages.